Trappist/src/Promact.Trappist.Test/Tests/TestsRepositoryTest.cs

```diff
@@ -375,10 +333,171 @@ private DomainModel.Models.TestConduct.TestAttendees TestAttendee()
             var categoryAcList = Mapper.Map<List<DomainModel.Models.Category.Category>, List<CategoryAC>>(categoryList);
             //Creating Test
             var test = CreateTest("Maths");
-            await _testRepository.CreateTestAsync(test);            
+            await _testRepository.CreateTestAsync(test);
             await _testRepository.AddTestCategoriesAsync(test.Id, categoryAcList);
             var testAc = await _testRepository.GetTestByIdAsync(test.Id);
             Assert.Equal(1, testAc.CategoryAcList.Count());
         }
+        #endregion
+
+        #region Duplicate Test
+        /// <summary>
+        /// Duplicates a test and save it in database
+        /// </summary>
+        [Fact]
+        public async Task DuplicateTest()
+        {
+            var category1 = CreateCategory("Aptitude");
+            var questiontoCreate1 = "Question1";
+            var question1 = CreateSingleAnswerQuestion(category1, questiontoCreate1);
+            var category2 = CreateCategory("Logical");
+            var questiontoCreate2 = "Question2";
+            var question2 = CreateSingleAnswerQuestion(category1, questiontoCreate2);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused 'question2' local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1481)